packages/expo-localization/android/src/main/java/expo/modules/localization/LocalizationPackage.kt

```diff
@@ -0,0 +1,10 @@
+package expo.modules.localization
+
+import android.content.Context
+import expo.modules.core.BasePackage
+import expo.modules.core.ExportedModule
+
+class LocalizationPackage : BasePackage() {
+  override fun createExportedModules(context: Context): List<ExportedModule> =
+    listOf<ExportedModule>(LocalizationModule(context))
+}
```

Comment:
```suggestion
class LocalizationPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> =
    listOf(LocalizationModule(context))
}
```
Type should be automatically inferred, as return type is stated explicitly (? something to verify)